test/unit/org/apache/cassandra/db/compaction/unified/ControllerTest.java

```diff
@@ -121,6 +121,50 @@ public void testValidateOptionsIntegers()
         testValidateOptions(true);
     }
 
+    @Test
+    public void testValidateOptionsInvalidTargetSSTableSize()
+    {
+        //Test 1: TARGET_SSTABLE_SIZE_OPTION = 12E899, the value reported in CASSANDRA-20398 
+        try {
+            Map<String, String> options = new HashMap<>();
+            options.putIfAbsent(Controller.TARGET_SSTABLE_SIZE_OPTION, "12E899");
+            Controller.validateOptions(options);
+            fail("TARGET_SSTABLE_SIZE_OPTION = 12E899 should have caused exception");
+        } catch(ConfigurationException e) {
+        }
```

Comment:
please remove the try/catch. 


https://cassandra.apache.org/_/development/code_style.html

>Never ever write catch (…) {} or catch (…) { logger.error() } merely to satisfy Java’s compile-time exception checking.

Reply:
done. Thanks